connector/src/main/java/org/geysermc/connector/network/translators/bedrock/entity/player/BedrockActionTranslator.java

```diff
@@ -244,6 +240,7 @@ public void translate(PlayerActionPacket packet, GeyserSession session) {
                 break;
             case JUMP:
                 // Leaving as a potential placeholder for an event or soul sand fixing
+                entity.setOnGround(false); // Increase block break time while jumping
```

Comment:
Is this set to true when the player lands?

Reply:
I'm pretty sure that is handled by `BedrockMovePlayerTranslator`.